content/_index.md

```
@@ -5,10 +5,6 @@ nodate: true
 nofeed: true
 ---
 
-My name is Harper Reed, and this is my blog. If you want to know more about me, visit my website: [harper.lol](https://harper.lol).
+### My name is Harper Reed, and this is my blog. If you want to know more about me, visit my website: [harper.lol](https://harper.lol).
```

Markdownlint (0.35.0)

Punctuation: '.'
Trailing punctuation in heading

(MD026, no-trailing-punctuation)